modules/brands/composables/useBrand.ts

```diff
@@ -26,10 +26,6 @@ export default (brand: Ref<ReturnType<typeof ToBrand>>) => {
       title: 'Brands',
       link: localePath('brands'),
     },
-    {
```

Comment:
Also remove the `Home` breadcrumb while you are at it so its consistent with the rest.